app/uk/gov/hmrc/gform/gform/GovukExtensions.scala

```diff
@@ -0,0 +1 @@
+
```

Comment:
Shouldn't this file be removed?

Reply:
I dont see this file in my branch. When I rebased from master, this did show up as deleted

Reply:
For some reason search didnt show it. I have removed it now